ci/docker-compose.yml

```diff
@@ -9,6 +9,15 @@ services:
     environment:
       POSTGRES_PASSWORD: ''
 
+  mssql:
+    image: mcr.microsoft.com/mssql/server:2017-latest
+    hostname: mssql
+    ports:
+      - 1433:1433
+    environment:
+      ACCEPT_EULA: Y
+      SA_PASSWORD: Rootm3plz
```

Comment:
```
ERROR: Unable to set system administrator password: Password validation failed. The password does not meet SQL Server password policy requirements because it is not complex enough. The password must be at least 8 characters long and contain characters from three of the following four sets: Uppercase letters, Lowercase letters, Base 10 digits, and Symbols..
```

I think the mssql server is not starting because you are not using any symbol for the password .. but maybe I am wrong.

but just in case, could you change the password to something like `Ibis_MSSQL_2017`

Reply:
Done.